### ISSUE TITLE
libedgetpu1-max version in docker.base should be locked

### ISSUE THREAD

**Issue by no2chem** [blakeblackshear/frigate#289]
**Describe the bug**
Currently, the version of libedgetpu1-max is not locked. Unfortunately, Google often releases breaking changes between libedgetpu and tflite_runtime, requiring them to be in sync. In November, Google released libedgetpu 15, which breaks with the locked version tflite_runtime 2.1. 

This error manifests itself as an unsupported datatype error,
```
RuntimeError: Internal: Unsupported data type in custom op handler: 141308532Node number 2 (EdgeTpuDelegateForCustomOp) failed to prepare.
```

This causes the current docker containers (built today) to break, and introduces a divergence between docker containers built now and last month.

**Version of frigate**
0.7.4

**Config file**
N/A

**Logs**
```
RuntimeError: Internal: Unsupported data type in custom op handler: 141308532Node number 2 (EdgeTpuDelegateForCustomOp) failed to prepare.
```

**Computer Hardware**
 - OS: Ubuntu 20.04
 - Virtualization: Docker
 - Coral Version: USB

**Additional context**
I think the fix would be to lock the version of libedgetpu

**Comment by blakeblackshear** [comment]
I merged your PR, but it doesn't look like libedgetpu1 has a version 15.0. The latest I see is 14.1. Was it pulled?

**Comment by blakeblackshear** [comment]
I see it in unstable: https://packages.cloud.google.com/apt/dists/coral-edgetpu-unstable/main/binary-amd64/Packages

**Comment by no2chem** [comment]
@blakeblackshear strange, I wonder if it was pulled or something? It was pulling it all weekend

**Comment by blakeblackshear** [comment]
Im going to leave it and hope it comes back. Most people are pulling pre-built images.

**Comment by no2chem** [comment]
It seems like it was probably pulled, from the docker I built this weekend:

```

cat /etc/apt/sources.list.d/coral-edgetpu.list
deb https://packages.cloud.google.com/apt coral-edgetpu-stable main
```

```
apt-cache showpkg libedgetpu1-max
Package: libedgetpu1-max
Versions:
15.0 (/var/lib/dpkg/status)
 Description Language:
                 File: /var/lib/dpkg/status
                  MD5: f1ee4496b3fdc005b2849021c9f3c9e9


Reverse Depends:
Dependencies:
15.0 - libc6 (0 (null)) libgcc1 (0 (null)) libstdc++6 (0 (null)) libusb-1.0-0 (0 (null)) debconf (18 0.5) debconf-2.0 (0 (null)) libedgetpu1 (0 (null)) libedgetpu1-legacy (0 (null))
Provides:
15.0 - libedgetpu1 (= 15.0)
Reverse Provides:
```

I did notice a new `python3-tflite-runtime` package in unstable too. Might be worth using that once it makes stable too.

**Comment by scottgrobinson** [comment]
Just to add to what no2chem said - I spent a number of hours debugging this on Saturday/Sunday. From what I can tell, I think 15 was pushed to stable, then removed. So when trying to work out why my main OS was working fine and I couldn't get my docker container to play ball nicely - I spent hours and hours going around in circles trying to work out why the hell the newer versions was on the OS only!

In short - no2chem isn't going crazy and I think pinning would be handy to prevent this :)
